src/chainlit/frontend/src/components/element/audio.tsx

```diff
@@ -1,14 +1,31 @@
+import { Box, useTheme } from '@mui/material';
+import { grey } from 'palette';
 import { IAudioElement } from 'state/element';
 
 interface Props {
   element: IAudioElement;
 }
 
 export default function AudioElement({ element }: Props) {
+  const theme = useTheme();
   if (!element.url && !element.content) {
     return null;
   }
   const className = `${element.display}-audio`;
   const src = element.url || URL.createObjectURL(new Blob([element.content!]));
-  return <audio controls src={src} className={className}></audio>;
+  return (
```

Comment:
it is better to use the Typography element here like 

```jsx
      <Typography
        color="text.primary"
        sx={{ fontSize: '14px', fontWeight: 700 }}
      >...</Typography>
```

unless the color is different from the main or secondary color in the theme?

Reply:
I've used a color that's different from the main/secondary color. It's slightly muted, like the Figma design of the image filename (that's the closest thing I've found for this purpose).

<img width="1290" alt="image" src="https://github.com/Chainlit/chainlit/assets/494686/f5fe7d2d-6ae1-47eb-8e42-580b0c6efb63">